app/models/linkset.rb

```diff
@@ -12,6 +12,14 @@ def empty?
   end
 
   def update_attributes_from_gem_specification!(spec)
-    self.update_attributes!(:home => spec.homepage)
+    meta = spec.metadata || {}
+    self.update_attributes!({
+      :home => spec.homepage,
+      :wiki => meta["wiki_uri"],
+      :docs => meta["documentation_uri"],
+      :mail => meta["mailing_list_uri"],
+      :code => meta["source_code_uri"],
+      :bugs => meta["bug_tracker_uri"]
+    }.delete_if { |_n, v| v.nil? })
```

Comment:
@bf4 This started from #718 - there are a lot of possibilities here and a lot of different projects that need to be updated.

Since we return these keys in the api, this is allowing developers to populate it from the gemspec metadata. You can see api defined in [rubygems](https://github.com/rubygems/rubygems.org/blob/63d43b660aa8430320c0db430b15e2def306e3ac/app/models/rubygem.rb#L162-L167).